src/js/stores/document.js

```diff
@@ -813,8 +812,17 @@ define(function (require, exports, module) {
          * NOTE: Assumes that each layer now only has a single text style,
          * and adjusts the model accordingly.
          *
+         * example payload:
+         * {
```

Comment:
Can you jsdoc this according to object jsdoc rules? `@param {number} payload.documentID` etc.